SPECS/kexec-tools/kdump.conf

```diff
@@ -179,7 +179,7 @@ core_collector makedumpfile -l --message-level 1 -d 31
 #extra_modules gfs2
 #failure_action shell
 #force_rebuild 1
-force_no_rebuild 1
+#force_no_rebuild 1
```

Comment:
since this is changing a config(noreplace) file, when existing instances upgrade to this version, if they've modified this config file themselves then this change won't get applied. When that is combined with the kdumpctl change to the initrd path, this could result in the kdump.service failing to start, because no kdump initrd exists (i.e. an explicit kdump rebuild would be needed to create it, or manually editing this config file on each instance).

Reply:
Good catch. I'm considering a change into kdumpctl to adjust the TARGET_INITRD path based on the force_no_rebuild value, so the proper TARGET_INITRD is used:
- if `force_no_rebuild 1`, `TARGET_INITRD="${KDUMP_BOOTDIR}/initrd.img-${kdump_kver}"`
- if `force_no_rebuild 0` -> `TARGET_INITRD="${KDUMP_BOOTDIR}/initrd.img-${kdump_kver}kdump`
 
And remove this change in Mariner 3.0 wholesale.
This change may be better served as a patch file to kexec-tools, rather than a direct edit into kdumpctl.